### PR TITLE
Feat/navigation

### PR DESCRIPTION
**Pull request by biswanath-tewari**
features added
1. mockbee setup
2. context setup
3. auth screens
4. navigation setup

### CODE REVIEW

File: src/navigation/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const LoggedInRoute = () => {
+  return <div>Product list</div>
+}
+const LoggedOutRoutes = () => {
+  return (
+    <Routes>
+      <Route path="/" exact element={<LandingPage />} />
+      <Route path="/login" element={<LoginPage />} />
+      <Route path="/signup" element={<SignupPage />} />
+      <Route path="/shop" element={<LoginPage />} />
```
Comment (SukanyaSen530) [inline src/navigation/index.js]: `/login` -- either route is wrong or if it is product listing page then shop should be in public routes.

**Review by SukanyaSen530 — APPROVED** [review]
Followed good coding practices. Systematic and clean code. :)

File: src/pages/Signup/styles.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  margin: 1rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.form-checkbox {
+  margin-right: 0.5rem;
+  cursor: pointer;
```
Comment (sharathnair9999) [inline src/pages/Signup/styles.scss:45]: you can use utility class for pointer and generic margin and padding values

File: src/pages/Login/styles.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
+.main {
+  min-height: 90vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (sharathnair9999) [inline src/pages/Login/styles.scss:5]: common flex properties can also be wrapped as utility

**Review by sharathnair9999 — APPROVED** [review]
Apart from using few utilities in CSS, everything looks good

File: src/pages/Signup/styles.scss (hunk 2 above)
Comment (biswanath-tewari) [reply src/pages/Signup/styles.scss:45]: thanks for the suggestion, I will add this utility to my CL. And this margin value is situation-specific.

File: src/pages/Login/styles.scss (hunk 3 above)
Comment (biswanath-tewari) [reply src/pages/Login/styles.scss:5]: Yes, I have been meaning to add these utilities once am done with my CL's review.

File: src/navigation/index.js (hunk 1 above)
Comment (biswanath-tewari) [reply src/navigation/index.js]: corrected it. Thanks for the feedback.

**Pull request merged**